core/templates/domain/collection/guest-collection-progress.service.spec.ts

```diff
@@ -195,6 +195,13 @@ describe('Guest collection progress service', () => {
         guestCollectionProgressService.getCompletedExplorationIds(collection));
       expect(completedIds).toEqual([_expId0, _expId2]);
     });
+
+    it('should throw error if collection id is invalid', () => {
+      var collection = _createCollection(null, '');
+      expect(() => {
+        guestCollectionProgressService.getCompletedExplorationIds(collection);
+      }).toThrowError();
```

Comment:
shouldn't the error message be specified here?

Reply:
done